include/dd/ComplexTable.hpp

```diff
@@ -380,52 +416,54 @@ namespace dd {
                << ", gc calls: " << gcCalls
                << ", gc runs: " << gcRuns
                << "\n";
-            //clang-format on
+            // clang-format on
             return os;
         }
 
-        // table lookup statistics
-        std::size_t collisions = 0;
-        std::size_t insert_collisions = 0;
-        std::size_t hits = 0;
-        std::size_t find_or_inserts = 0;
-        std::size_t lookups = 0;
-        std::size_t inserts = 0;
-        std::size_t lower_neighbors = 0;
-        std::size_t upper_neighbors = 0;
-
     private:
-        using Bucket = Entry *;
-        using Table = std::array<Bucket, NBUCKET>;
+        using Bucket = Entry*;
+        using Table  = std::array<Bucket, NBUCKET>;
 
         Table table{};
 
+        std::array<Entry*, NBUCKET> last_entry_table{};
```

Comment:
```suggestion
        std::array<Entry*, NBUCKET> lastEntryTable{};
```
what about naming this `tailTable`?

Reply:
Good suggestions!